modular_skyrat/modules/cargo/code/packs.dm

```diff
@@ -159,15 +159,11 @@
 	)
 	crate_name = "armor crate"
 
-/datum/supply_pack/security/helmets
+/datum/supply_pack/security/helmets_skyrat
 	name = "Helmets Crate"
-	desc = "Contains three standard-issue brain buckets. Requires Security access to open."
+	desc = "Contains three standard-issue brain buckets."
```

Comment:
Why are we removing the string about requiring Security access to open?

Reply:
Somewhere in the code it checks if the access is required and will add it to desc. Current version just doubles that "required" text.

Reply:
Alright, cheers!